dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.health.javax-4.0.feature

```diff
@@ -0,0 +1,7 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.org.eclipse.microprofile.health.javax-4.0
+singleton=true
+-bundles=io.openliberty.org.eclipse.microprofile.health.4.0.javax; location:="lib/"
```

Comment:
At GA, will this bundle move to dev/api/spec?

Reply:
Nope. This is pretty much internal use. It starts the javaxified bundle of the MP 4.0 API (needed for the `io.openliberty.microprofile.health.4.0.internal` bundle during runtime when its implements the classes/annotation of the API).

Customers/users are able to use the original (i.e. jakarta-ed) MP 4.0 API in the `dev/api/stable` or by pulling the API jar from maven (or downloading it from the project site)